sdk/communication/azure-communication-sms/tests/acs_sms_test_case.py

```diff
@@ -0,0 +1,33 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import os
+from devtools_testutils import AzureRecordedTestCase, is_live
+from azure.communication.sms._shared.utils import parse_connection_str
+
+class ACSSMTestCase(AzureRecordedTestCase):
```

Comment:
Please fix the typo, I believe it should say ACSSMSTestCase

Reply:
@phermanov-msft thanks for noticing it. Fixed